doc-content/enterprise-only/openshift/ref-custom-resource-parameters.adoc

```diff
@@ -0,0 +1,70 @@
+|`enableBusinessCentral` | -- | Enables integration with {CENTRAL} by configuring {CENTRAL} and Dashbuilder automatically. Only available on operator | True
+|`enableKieServer` | -- | Enables integration with {KIE_SERVER} by configuring {KIE_SERVER} and Dashbuilder automatically. Only available on operator. | True
+|`externalCompDir` | `DASHBUILDER_EXTERNAL_COMP_DIR` | Sets the base directory where dashboard ZIP files are stored. If `PersistentConfigs` is enabled and `ExternalCompDir` is not set to an existing path, the `/opt/kie/dashbuilder/components` directory is used. | --
+|`importFileLocation` | `DASHBUILDER_IMPORT_FILE_LOCATION` | Sets a static dashboard to run automatically. If this property is set, imports are not allowed. | --
+|`importsBaseDir` | `DASHBUILDER_IMPORTS_BASE_DIR` | Sets the base directory where dashboard ZIP files are stored. If `PersistentConfigs` is enabled and `ImportsBaseDir` is not set to an existing path, the `/opt/kie/dashbuilder/imports` directory is used. If `ImportFileLocation` is set `ImportsBaseDir` is ignored. | --
+|`kieServerDataSets` | `KIESERVER_DATASETS` | Defines the KIE Server Datasets access configuration, for example:
+
+`name`: dataset name +
+`location`: dataset address, url +
+`password`: dataset password (leave blank if using a token) +
+`user`: dataset username (leave blank if using a token) +
+`replaceQuery`: Enables automatic query creation if you use dashboards on a {KIE_SERVER}. +
+`token`: {KIE_SERVER} token (leave blank if using user and password
+
+For multi datasets, use comma separated values. Dashes are replaced by underscore on environment variable names. The valid values for this prefix are:
+
+${datasetPrefix}_LOCATION, ${datasetPrefix}_USER, ${datasetPrefix}_PASSWORD, ${datasetPrefix}_TOKEN, ${datasetPrefix}_REPLACE_QUERY.
+
+For more information, see https://docs.jboss.org/jbpm/release/7.45.0.Final/jbpm-docs/html_single/#_kie_server_integration. | --
+|`kieServerTemplates` | `KIESERVER_SERVER_TEMPLATES` | Defines the KIE Server Templates access configuration, for example:
+
+`name`: dataset name +
+`location`: dataset address, url +
+`password`: dataset password (leave blank if using a token) +
+`user`: dataset username (leave blank if using a token) +
+`replaceQuery`: Enables automatic query creation if you use dashboards on a {KIE_SERVER}. +
+`token`: {KIE_SERVER} token (leave blank if using user and password
+
+For multi serverTemplate, use comma separated values. Dashes are replaced by underscore on environment variable names. The valid values for this prefix are:
+
+${serverTemplatePrefix}_LOCATION, ${serverTemplatePrefix}_USER, ${serverTemplatePrefix}_PASSWORD, ${serverTemplatePrefix}_TOKEN, ${datasetPrefix}_REPLACE_QUERY.
+
+For more information, see https://docs.jboss.org/jbpm/release/7.45.0.Final/jbpm-docs/html_single/#_kie_server_integration. | --
```

Comment:
imho this should point for the product link on the official doc.

Reply:
Will replace with {URL_INSTALLING_AND_CONFIGURING}#dashbuilder-runtimes-con[_Dashbuilder runtimes_] in the _{INSTALLING_ON_EAP}_ guide.